### PR TITLE
Fix profiler trying to serialize invalid utf8

### PR DESCRIPTION
**Pull request by CarlSchwan**
The cookie value contains invalid utf8 characters most of the time so
let's just ignore it as it is also not that interesting to analyse.

### CODE REVIEW

**Review by blizzz — APPROVED** [review]

**Comment by blizzz** [comment]
The cookie is to be treated as an opaque structure by RFC. It is only created and interpreted by the server implementation. On client side the job is only passing it forth and back with the requests. A mechanism to keep track/state on the LDAP server.

File: apps/user_ldap/lib/LDAP.php [hunk 1]
```diff
@@ -320,7 +320,15 @@ private function preFunctionCall(string $functionName, array $args): void {
 		$this->curArgs = $args;
 
 		if ($this->dataCollector !== null) {
-			$args = array_map(fn ($item) => (!$this->isResource($item) ? $item : '(resource)'), $this->curArgs);
+			$args = array_map(function ($item) {
+				if ($this->isResource($item)) {
+					return '(resource)';
+				}
+				if (isset($item[0]['value']['cookie'])) {
+					$item[0]['value']['cookie'] = "";
```
Comment (come-nc) [inline apps/user_ldap/lib/LDAP.php:328]:
```suggestion
					$item[0]['value']['cookie'] = "*opaque cookie*";
```
As an empty cookie has a special meaning in some cases?

Comment (come-nc) [reply apps/user_ldap/lib/LDAP.php:328]: Well you also need to change the if to only replace non-empty strings if you go for this.

Comment (CarlSchwan) [reply apps/user_ldap/lib/LDAP.php:328]: sounds like a good idea

**Review by come-nc — APPROVED** [review]

**Pull request merged**